hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -13,7 +13,6 @@
     ZOOKEEPER_BASE_SLEEP_TIME("zookeeper.base.sleep.time", 1000),
     ZOOKEEPER_CONNECTION_TIMEOUT("zookeeper.connection.timeout", 10000),
     ZOOKEEPER_SESSION_TIMEOUT("zookeeper.session.timeout", 10000),
-    ZOOKEEPER_SYNC_TIME("zookeeper.sync.time", 20000),
```

Comment:
Previous value was set to 20k, current is 4k but there will be more retries. We will see how it performs, finally we can change those values.